app/views/event/index.html.erb

```diff
@@ -0,0 +1,56 @@
+<h1>Event#index <%= link_to  "Create Event", events_new_path , class: "btn btn-primary"%>
+<%= link_to  "My Profile", profile_path , class: "btn btn-primary"%>
+</h1> 
+
+<h3>Upcoming</h3>
+<div class="events-holder">
+    <% @events.upcoming.each do |event| -%>        
+    <div class="card card-mb" style="width: 25%;">
+      <img class="card-img-top" src="https://via.placeholder.com/180.png/09f/fff" alt="Card image cap">
+      <div class="card-body">
+        <h5 class="card-title"> <%= event.title %> </h5>
+        <p class="card-text"> description : <%= event.description %></p>
+        <p class="card-text"> Event Date : <%= event.date %></p>
+        <p class="card-text"> Number of attendees : <%= event.EventAttendance.count %></p>
+        <p class="card-text"> created at : <%= event.created_at.to_date %></p>
+
+       <%= link_to  "View", events_path + '/' + event.id.to_s , class: 'btn btn-primary'%>
```

Comment:
- [ ] There should be no logic in views. -- Rails has built-in mechanism to deal with view-related logic which is View Helpers (app/helpers/*_helper.rb). So the common convention is to move any logic in the views to the helpers. Please take a look at [this](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers) for more information.


![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png)